stash-cache-origin/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:centos7
```

Comment:
If you base this off of `opensciencegrid/xcache:development` you'll get the fetch-crl and xcache-reporter crons for free

Reply:
How will this be handled as we evolve from development to production versions?
Path the Dockerfile in the production branch?

Reply:
I still have to write this all up but the plan is to not use branches to separate out dev/testing/prod but instead to have master push to dev if they pass automated tests, then manually tag testing/prod versions after some amount of acceptance testing.

With that model, I think we can just always use the development tag.